src/login/translation/messageIndex.js

```diff
@@ -46,15 +46,20 @@ export const unformattedMessages = defineMessages({
     defaultMessage: "Code is formatted as five groups of characters and numbers, separated by hyphens",
     description: "Help text for email confirmation code",
   },
-  "emails.invalid_code": {
-    id: "emails.invalid_code",
+  "modal.long_code_invalid_format": {
+    id: "modal.long_code_invalid_format",
     defaultMessage: `Invalid confirmation code`,
     description: "error text for invalid confirmation code"
   },
-  "mobile.confirm_code_wrong_length": {
-    id: "mobile.confirm_code_wrong_length",
-    defaultMessage: `Code must be 10-12 characters long, only letters and numbers allowed`,
-    description: "Error message for wrong length",
+  "modal.short_code_invalid_format": {
+    id: "modal.short_code_invalid_format",
+    defaultMessage: `The code is 10 characters long`,
+    description: "Error message for mobile and letter wrong length",
+  },
+  "modal.security_code_invalid_format": {
```

Comment:
Perhaps this could be renamed to something more like `"securitykey_description_invalid_format"`, to specify that it relates to security keys rather than just "security" and to the description of security keys rather than a "code".